docker-compose.yml

```diff
@@ -21,8 +21,11 @@ services:
   db:
     image: postgres:14.2-alpine3.15
     restart: always
+    user: postgres
     volumes:
       - transcendence-db:/var/lib/postgresql/data
+      - ./db/fill-db.sh:/docker-entrypoint-initdb.d/fill-db.sh
```

Comment:
https://levelup.gitconnected.com/creating-and-filling-a-postgres-db-with-docker-compose-e1607f6f882f